### DISCUSSION TITLE
Open a JSON and then loop through all the values

### DISCUSSION THREAD

**Discussion by SpaceR3** [nlohmann/json#2323] [Q&A] [answered]
Hey, I have been looking through the docs for 10 minutes now. I can't find how to open a JSON file and JSON it.
--(FIXED ON OPENING FILES)
--(STILL NEED HELP ON THE LAST PART)
I have a JSON file like this:

```
{
"something": "some"
},
{
"something": "som2"
},
{
"something": "som3"
}
```

I need to open it and then loop through all the values.

**Comment by SpaceR3** [comment, marked as answer]
Got it working

**Comment by nlohmann** [comment]
Parsing is described in https://github.com/nlohmann/json#serialization--deserialization - you can pass `std::string` or `std::ifstream` to function `json::parse`.

Iterating is described in https://github.com/nlohmann/json#stl-like-access - JSON objects behave like `std::map`.
